docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -80,6 +80,22 @@ property, which can be set to ``UNNECESSARY`` (the default),
 
 .. include:: jdbc-type-mapping.fragment
 
+.. _sqlserver-sql-support:
```

Comment:
mysql, not sqlserver

Reply:
I corrected this while you were reviewing. Already amended.